astropy/coordinates/stokes_coord.py

```diff
@@ -0,0 +1,238 @@
+from collections import namedtuple
+from contextlib import contextmanager
+from copy import copy
+from typing import Dict
+
+import numpy as np
+
+from astropy.utils import unbroadcast
+from astropy.utils.data_info import MixinInfo
+from astropy.utils.shapes import ShapedLikeNDArray
+
+__all__ = ["StokesCoord", "custom_stokes_symbol_mapping", "StokesSymbol"]
+
+
+StokesSymbol = namedtuple("StokesSymbol", ["symbol", "description"], defaults=[""])
+
+# This is table 29 in the FITS 4.0 paper
+FITS_STOKES_VALUE_SYMBOL_MAP = {
+    1: StokesSymbol("I", "Standard Stokes unpolarized"),
+    2: StokesSymbol("Q", "Standard Stokes linear"),
+    3: StokesSymbol("U", "Standard Stokes linear"),
+    4: StokesSymbol("V", "Standard Stokes circular"),
+    -1: StokesSymbol("RR", "Right-right circular"),
```

Comment:
I am happy for you or someone else with more understanding to suggest better descriptions. I copied these out of the FITS 4 paper.

Reply:
I'm fairly sure the above is correct, and it would be similar for `XY`. But I'm also fine with just sticking to what the table states, or perhaps add the mathematical implications after?
```
    -5: StokesSymbol("XX", "X parallel linear: <XX*>"),
    -6: StokesSymbol("YY", "Y parallel linear: <YY*>"),
    -7: StokesSymbol("XY", "XY cross linear: Re(<XY*>)"),
    -8: StokesSymbol("YX", "YX cross linear: Re(<YX*>"),
```